src/test/resources/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImplGetCaseByUprnTest.AddressIndexAddressCompositeDTO.json

```diff
@@ -9,5 +9,6 @@
   "foundAddressType": "PAF",
   "censusAddressType": "HH",
   "censusEstabType": "Household",
-  "countryCode": "E"
+  "countryCode": "E",
+  "organisationName": "The AI Address Composite Org"
```

Comment:
please add new line to end of JSON file (remove github red circle warning) for consistency

Reply:
done